.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This doesn't need to be in here. Things in the .vscode/ directory are usually gitignored. Just need to delete this file and put .vscode/ in the gitignore.